src/brs/services/impl/ParameterServiceImpl.java

```diff
@@ -52,11 +52,24 @@ public ParameterServiceImpl(AccountService accountService, AliasService aliasSer
   @Override
   public Account getAccount(HttpServletRequest req) throws BurstException {
     String accountId = Convert.emptyToNull(req.getParameter(ACCOUNT_PARAMETER));
+    String heightValue = Convert.emptyToNull(req.getParameter(HEIGHT_PARAMETER));
     if (accountId == null) {
       throw new ParameterException(MISSING_ACCOUNT);
     }
     try {
-      Account account = accountService.getAccount(Convert.parseAccountId(accountId));
+      Account account = null;
+      if (heightValue != null) {
+    	try {
+    	  int height = Integer.parseInt(heightValue);
+    	  if (height < 0 || height > blockchain.getHeight())
+    		throw new ParameterException(INCORRECT_HEIGHT);
+      	  account = accountService.getAccount(Convert.parseAccountId(accountId), height);
```

Comment:
could you fix pls indentations in this section. in another branch I added editorconfig, most IDE support it, sometimes need install plugin, intellij supports this by default. I see above indentations 4 spaces, below 2, with editorconfig under all IDE will one codestyle.

Reply:
Thanks for the heads up, this is an experimental fork (it is breaking some stuff right now). I'll add your editorconfig file.